src/lib/components/commons/FilterPanel/FilterPanel.styled.ts

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import { StyledButton } from '../Button/Button.styled';
+
+type IStyledFilterProps = {
+justify?: string;
+width?: string; 
+isButtonGroup?: boolean;
+}
+
+export const StyledFilterPanel = styled.div`
+            display: grid;
+            grid-column: 1/-1;
+            gap: .5rem;
+            position: absolute;
+            top:15%;
+            background-color:#ecf0f6;
+            z-index:1;
+            border-radius: 10px;
+            padding:2%;
+            box-shadow:var(--box-shadow-1);
+`;
+
+export const StyledFilterControlDiv= styled.div<IStyledFilterProps>`
+display:flex;
+flex-wrap: wrap;
+gap:2%;
```

Comment:
what is this percentage property? why not using rem or pixels?

Reply:
im actually used to use precentage. 
I needed to pay attention  to your use of rem...